extension/lib/position.js

```diff
@@ -1,52 +1,50 @@
 let $ = require('jquery');
 
-let isImage = function(element) {
-  return element.prop('tagName') === 'IMG';
-};
-
-let setPositionValue = function(position, key, value) {
+let setObjectValue = function(object, key, value) {
   if (value) {
-    position[key] = value;
+    object[key] = value;
   }
 
-  return position;
+  return object;
 };
 
-let constructPosition = function(callback) {
+let construct = function(callback) {
   let url      = 'https://localhost/api/v1/selectors';
   let promise  = $.getJSON(url, {
     host: location.host
   });
 
   promise.done(function(json, status, xhr) {
-    let position = Object.keys(json.data).reduce(function(position, key) {
-      let elements = $(json.data[key]);
+    let object = Object.keys(json.data).reduce(function(object, key) {
```

Comment:
For the selector resource, aren't the keys the css selectors? It looks like the client wants keys in `json.data[key]` to return an object with a key, `selector`, which will contain the css selector, and another key, `attribute`, which I assume will return the thing that we want the selector to grab.